src/snc/agents/hedgehog/strategic_idling/strategic_idling.py

```diff
@@ -237,23 +245,34 @@ def _get_vector_defining_possible_idling_direction(w_star: WorkloadSpace,
     def _create_find_workload_with_min_eff_cost_by_idling_lp_program(self):
         x_var = cvx.Variable((self._num_buffers, 1), nonneg=True)  # Variable
         w_par = cvx.Parameter((self._num_bottlenecks, 1))  # Parameter
+        safety_stocks_vec = cvx.Parameter((self._num_bottlenecks, 1))
         penalty_coeff_w_star = self.strategic_idling_params.penalty_coeff_w_star
         objective = cvx.Minimize(
             self._cost_per_buffer.T @ x_var
             + penalty_coeff_w_star * cvx.sum(self._workload_mat @ x_var - w_par))
         constraints = [self._workload_mat @ x_var >= w_par]
+        a_mat = np.vstack(self.list_boundary_constraint_matrices)
```

Comment:
Could we rename `a_mat` more descriptively, maybe something like `resource_to_buffer_mat`?